shared/teams/common/channel-hooks.tsx

```diff
@@ -47,7 +47,18 @@ export const useAllChannelMetas = (
       emptyArrForUseSelector
   )
   // TODO: not always a new map?
-  return new Map(filterNull(conversationMetas).map(a => [a.conversationIDKey, a]))
+  const channelMetas = new Map(filterNull(conversationMetas).map(a => [a.conversationIDKey, a]))
+
+  const [loadingChannels, setLoadingChannels] = React.useState(true)
```

Comment:
hmm I did this rather than just returning the result from `anyWaiting` because with that approach there's a period at the beginning where you don't appear to be loading. I fear the approach you suggest would have the same issue. WDYT?

Reply:
it should have the same behavior if it's initialized to `true`. 
Mount -> `loadingChannels` is `true` -> line above `getConversations` sets it to `true` -> callbacks set it to `false`

Reply:
Why call it set(true) if it's already true?

Reply:
I understand this comment now. I'm not sure why I did it this way.